src/main/java/org/opentestsystem/ap/ims/service/export/PdfItemExportServiceImpl.java

```diff
@@ -217,14 +217,14 @@ public FileVisitResult preVisitDirectory(Path dir, BasicFileAttributes attrs) th
     }
 
     private void moveImages(Path clonedItemDirectory) {
-        for(String icon : images) {
+        for (String icon : images) {
             ClassPathResource audioIcon = new ClassPathResource("static/images/" + icon);
 
             File newLocation = new File(clonedItemDirectory.toFile(), icon);
 
             try (InputStream fis = audioIcon.getInputStream();
                  FileOutputStream fos = new FileOutputStream(newLocation)) {
-                org.python.apache.commons.compress.utils.IOUtils.copy(fis, fos);
+                IOUtils.copy(fis, fos);
```

Comment:
Not sure why python utils was being used.  Removed and used the standard copy apache ioutils call.